demo/src/plugins/dashboard/components/active-edits/index.jsx

```diff
@@ -0,0 +1,107 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+import { connect } from 'react-redux';
+import Message from '@gdbots/pbj/Message';
+import { STATUS_FULFILLED, STATUS_PENDING } from '@triniti/app/constants';
+import {
+  Button,
+  Card,
+  CardBody,
+  CardHeader,
+  Icon,
+  StatusMessage,
+  Table,
+} from '@triniti/admin-ui-plugin/components';
+import delegate from './delegate';
+import selector from './selector';
+import TableRow from './TableRow';
+
+class ActiveEdits extends React.Component {
+    static propTypes = {
+      accessToken: PropTypes.string.isRequired,
+      collaborationNodes: PropTypes.arrayOf(PropTypes.instanceOf(Message)).isRequired,
+      contentType: PropTypes.oneOf(['articles', null]),
```

Comment:
@nanetteCodes can just make it PropTypes.string with a default

Reply:
updated